server/src/main/java/com/cloud/resource/RollingMaintenanceManagerImpl.java

```diff
@@ -439,7 +439,7 @@ private void enableClusterIfDisabled(Cluster cluster, Set<Long> disabledClusters
      */
     private boolean isMaintenanceStageAvoided(Host host, Map<Long, String> hostsToAvoidMaintenance, List<HostSkipped> hostsSkipped) {
         if (hostsToAvoidMaintenance.containsKey(host.getId())) {
-            s_logger.debug("Host " + host.getId() + " is not being put into maintenance, skipping it");
+            s_logger.debug(String.format("%s is not being put into maintenance, skipping it", host));
```

Comment:
I think this message could be more clear, e.g.: 
`%s is in avoid maintenance list [%s], skipping its maintenance...`